composer.json

```diff
@@ -19,10 +19,16 @@
             "": "src/"
         }
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/rubensayshi/http-signatures-php"
```

Comment:
this needs to be changed once https://github.com/99designs/http-signatures-php/pull/22 is merged (and before this PR can be merged)